plugins/woocommerce/tests/php/src/Internal/FraudProtection/ApiClientTest.php

```
@@ -480,4 +480,86 @@ function ( $preempt, $args ) use ( &$captured_request_body ) {
 		$this->assertArrayNotHasKey( 'billing_name', $decoded_body, 'Should filter out null billing_name' );
 	}
+
+	/**
+	 * @testdox Send Event should include prior_events in payload when provided.
+	 */
+	public function test_send_event_includes_prior_events_in_payload(): void {
+		$captured_request_body = null;
+
+		add_filter(
+			'pre_http_request',
+			function ( $preempt, $args ) use ( &$captured_request_body ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$preempt'. (undefined)

(UnusedFormalParameter)